### ISSUE TITLE
About Secondary Structure Datasets

### ISSUE THREAD

**Issue by Carrotljw** [QData/DeepVHPPI#3]
hi~  May I ask which is the test dataset for secondary structure prediction, I downloaded the dataset on tape, there are training sets and validation sets, but there are no test sets but csap12, cb513, ts115, which are in the test set. one or all of them, hope to get your reply~
best!😁

**Comment by jacklanchantin** [comment]
As done in the Rao et al paper:

>  We report accuracy on a per-amino acid basis on the CB513 [38] dataset.

**Comment by Carrotljw** [comment]
Thanks! 😁
